server/routes/index.ts

```diff
@@ -298,6 +298,34 @@ export function defineRoutes(router: IRouter, esClient: IClusterClient) {
       }
     }
   );
+
+  router.put(
+    {
+      path: `${API_PREFIX}/configuration/audit/config`,
+      validate: {
+        body: schema.any(),
```

Comment:
No body schema?

Reply:
Followed other examples in the same file. Also this is for configurations which are all optional.